normandy/recipes/exports.py

```diff
@@ -0,0 +1,207 @@
+import logging
+
+import kinto_http
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+logger = logging.getLogger(__name__)
+
+
+APPROVE_CHANGES_FLAG = {"status": "to-sign"}
+
+
+def recipe_as_record(recipe):
+    """
+    Transform a recipe to a dict with the minimum amount of fields needed for clients
+    to verify and execute recipes.
+
+    :param recipe: a recipe ready to be exported.
+    :returns: a dict to be posted on Remote Settings.
+    """
+    from normandy.recipes.api.v1.serializers import (
+        MinimalRecipeSerializer,
+    )  # avoid circular imports
+
+    serializer = MinimalRecipeSerializer(recipe)
+    record = serializer.data
+    record["id"] = str(recipe.id)
+    return record
+
+
+class RemoteSettings:
+    """
+    Interacts with a RemoteSettings service.
+
+    Basically, a recipe becomes a record in the dedicated collection on Remote Settings.
+    When it is disabled, the associated record is deleted.
+
+    Since Normandy already has the required approval/signoff features, the integration
+    bypasses the one of Remote Settings (leveraging a specific server configuration for this
+    particular collection).
+
+    .. notes::
+
+        Remote Settings signoff workflow relies on several buckets (see kinto-signer API).
+
+        The `main-workspace` is only readable and writable by authorized accounts.
+        The `main` bucket is read-only, but publicly readable. The Remote Settings
```

Comment:
If docstrings are restructured text this should be double backticks. (and the line above)